components/eam/cime_config/testdefs/testmods_dirs/eam/v3atm_rtmoff_mam5/user_nl_eam_cpcp

```diff
@@ -0,0 +1,77 @@
+ history_gaschmbudget_2D_levels = .false.
+ history_UCIgaschmbudget_2D = .false.
+ history_UCIgaschmbudget_2D_levels = .false.
+ gaschmbudget_2D_L1_s =  1
+ gaschmbudget_2D_L1_e = 26
+ gaschmbudget_2D_L2_s = 27
+ gaschmbudget_2D_L2_e = 38
+ gaschmbudget_2D_L3_s = 39
+ gaschmbudget_2D_L3_e = 58
+ gaschmbudget_2D_L4_s = 59
+ gaschmbudget_2D_L4_e = 72
+! history_aero_optics    = .true.
+! history_aerosol        = .true.
+! history_amwg           = .true.
+! history_budget         = .true.
+! history_verbose        = .true.
+ cosp_lite = .true.
+
+ linoz_psc_t = 198.0
+
+! update species for using F20TR-chemUCI-Linozv3
+! follow https://acme-climate.atlassian.net/wiki/spaces/NGDAP/pages/3566043211/20220914.PAN.MZThet.v2.LR.bi-grid.amip.chemUCI+Linozv3
+! MAM5 files
+ rad_climate            = 'A:H2OLNZ:H2O', 'N:O2:O2', 'N:CO2:CO2',
+         'A:O3:O3', 'A:N2OLNZ:N2O', 'A:CH4LNZ:CH4',
+         'N:CFC11:CFC11', 'N:CFC12:CFC12',
+         'M:mam5_mode1:$DIN_LOC_ROOT/atm/cam/physprops/mam5_mode1_rrtmg_aeronetdust_c141106.nc',
+         'M:mam5_mode2:$DIN_LOC_ROOT/atm/cam/physprops/mam5_mode2_rrtmg_c130628.nc',
+         'M:mam5_mode3:$DIN_LOC_ROOT/atm/cam/physprops/mam5_mode3_rrtmg_aeronetdust_c141106.nc',
+         'M:mam5_mode4:$DIN_LOC_ROOT/atm/cam/physprops/mam5_mode4_rrtmg_c130628.nc'
+         'M:mam5_mode5:$DIN_LOC_ROOT/atm/cam/physprops/mam5_mode5_rrtmg_sig1.2_dgnl.40_c03072023.nc'
+
+ drydep_srf_file                = '/lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mam/atmsrf_ne30pg2_200129.nc'
+ exo_coldens_file               = '/lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart/phot/exo_coldens.nc'
+ ext_frc_specifier              = 'SO2         -> /lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so2_elev_1850-2014_c180205.nc',
+         'SOAG        -> /lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_soag_elev_1850-2014_c180205.nc',
+         'bc_a4       -> /lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_bc_a4_elev_1850-2014_c180205.nc',
+         'num_a1      -> /lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a1_elev_1850-2014_c180205.nc',
+         'num_a2      -> /lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a2_elev_1850-2014_c180205.nc',
+         'num_a4      -> /lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a4_elev_1850-2014_c180205.nc',
+         'pom_a4      -> /lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_pom_a4_elev_1850-2014_c180205.nc',
+         'so4_a1      -> /lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so4_a1_elev_1850-2014_c180205.nc',
+         'so4_a2      -> /lcrc/group/e3sm/data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_so4_a2_elev_1850-2014_c180205.nc'
```

Comment:
I just noticed that the chrysalis paths are hardcoded here, shouldn't these paths use DIN_LOC_ROOT instead?